examples/billing/add_funds.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
+
+credit_card = EasyPost::CreditCard.fund('2000', 'primary')
```

Comment:
Can we do explicit parameters mapping in Ruby a la Python? Such as `(amount='2000', primary_or_secondary='primary')` (parameter names may be wrong)

Reply:
No